### PR TITLE
Update cheatSheets.md

### PR DESCRIPTION
**Pull request by Subathra19**
Added Arduino programming cheat sheet

### CODE REVIEW

File: cheatSheets.md [hunk 1]
```diff
@@ -48,3 +48,4 @@
 
 - [HTML Cheat Sheet](https://html.com/wp-content/uploads/html-cheat-sheet.pdf): This clear, easy-to-access PDF thoroughly explains HTML elements, with understandable explanations and examples.
 
+- [Arduion Programming Cheat Sheet](https://github.s3.amazonaws.com/downloads/liffiton/Arduino-Cheat-Sheet/Arduino%20Cheat%20Sheet.pdf):Arduion Programming Cheat Sheet covering all important syntax.
```
Comment (CodingSpecies) [inline cheatSheets.md:51]: @Subathra19 I think there is a little typo. Is `Arduion` meant to say Arduino? Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: cheatSheets.md (hunk 1 above)
Comment (MRslimcoder) [reply cheatSheets.md:51]: @CodingSpecies Good catch.

Comment (Subathra19) [reply cheatSheets.md:51]: @CodingSpecies  Thanks for the comment. Yes, its a typo

**Review by CodingSpecies — APPROVED** [review]
Looks good now! Thank you.

**Review by MRslimcoder — APPROVED** [review]
LGTM

**Comment by MRslimcoder** [comment]
@Subathra19 close this PR and send again build is failing.

**Comment by arjunkharbanda** [comment]
Sorry I had to close this the build was failing. Yoy can submit a new one if you want to

**Pull request closed without merging**